vdr/didweb/store_test.go

```diff
@@ -52,8 +52,6 @@ func Test_sqlStore_create(t *testing.T) {
 		verificationMethods, err := store.get(testDID)
 		require.NoError(t, err)
 		require.Len(t, verificationMethods, 2)
-		require.JSONEq(t, toJSON(vm1), toJSON(verificationMethods[0]))
```

Comment:
maybe we should sort them, otherwise DID documents might change if order is not guaranteerd>